sim/core/spell_resistances_test.go

```diff
@@ -121,7 +121,7 @@ func Test_PartialResistsVsBoss(t *testing.T) {
 		}
 
 		// no more level based resists
-		expectedAr := resist / (724 + resist)
+		expectedAr := resist / (991.5 + resist)
```

Comment:
If beta testing confirms that the mechanic is gone entirely for level 90 players, then `expectedAr` would actually be 0 right?